test/SadTest.cc

```diff
@@ -757,6 +755,54 @@ class SadLoopTest : public ::testing::WithParamInterface<SadLoopTestParam>,
             << search_area_height_ << "]";
     }
 
+};
+
+TEST_P(SadLoopTest, SadLoopSparseTest) {
+    check_sad_loop_sparse();
+}
+
+TEST_P(SadLoopTest, SadLoopTest) {
+    check_sad_loop();
+}
+
+INSTANTIATE_TEST_CASE_P(
+    LOOPSAD, SadLoopTest,
+    ::testing::Combine(::testing::ValuesIn(TEST_PATTERNS),
+                       ::testing::ValuesIn(TEST_SAD_SIZES),
+                       ::testing::ValuesIn(TEST_LOOP_AREAS)));
+/**
+ * @brief Unit test for SAD HME functions include:
+ *  - sad_loop_kernel_{sse4_1,avx2}_hme_l0_intrin
```

Comment:
Is there sad_loop_kerel_sse4_1_hme_l0_intrin function? If there is, seems like it's not being tested.

Reply:
It's in the test.